nucypher/blockchain/eth/token.py

```diff
@@ -200,3 +246,185 @@ def time_remaining(self, slang: bool = False) -> Union[int, str]:
         else:
             result = delta.seconds
         return result
+
+    #
+    # Validation
+    #
+
+    @staticmethod
+    def __handle_validation_failure(rulebook: Tuple[Tuple[bool, str], ...]) -> bool:
+        """Validate a staking rulebook"""
+        for rule, failure_message in rulebook:
+            if not rule:
+                raise ValueError(failure_message)
+        return True
+
+    def validate(self) -> bool:
+        return all((self.validate_value(), self.validate_duration()))
```

Comment:
`return self.validate_value() and self.validate_duration()` seems more readable to me

Reply:
Yes, there's a common preference in the codebase to use `all()` even with only two values (once I saw it with only one value!). I think it should be limited to >= 3 values.

Reply:
Yeah, this often happens because there were more than three checks in the past,m or might be more checks in the future.